package.json

```diff
@@ -35,33 +35,30 @@
 	],
 	"type": "module",
 	"engines": {
-		"node": ">= 14.16"
+		"node": ">= 16"
 	},
 	"scripts": {
-		"test": "xo && ava"
+		"test": "run-s test:*",
+		"test:compile": "tsc",
+		"test:quality": "xo",
+		"test:integration": "NODE_OPTIONS='--loader=tsx' ava"
 	},
 	"dependencies": {
-		"decompress": "^4.2.1",
-		"file-type": "^17.1.2",
-		"js-yaml": "^4.1.0",
-		"mammoth": "^1.4.21",
-		"pdf-parse": "^1.1.1",
-		"read-chunk": "^4.0.2",
-		"xlsx": "^0.18.5",
-		"xml2js": "^0.4.23"
+		"file-type": "^18.2.1",
+		"got": "12.6.0",
+		"mammoth": "1.6.0",
+		"pdf-parse": "1.1.1",
+		"pnpm": "8.6.1"
```

Comment:
should this be a runtime dependency?

Reply:
No, you're right, `pnpm` should not.

Wonder how that got there :0